### DISCUSSION TITLE
1st to 31st October Report - Furqan Ahmed

### DISCUSSION THREAD

**Discussion by nblogist** [ringdao-community-guild/discussion#195] [General]
# Work Summary for October 2024
This report continues the updates provided in the [previous report for September 2024](https://github.com/orgs/dcdao/discussions/182), as requested by the steering committee.

This month has been majorly focused on helping Darwinia get more connected and start going away from being a silo-ed; made a lot of people awe on seeing that Darwinia is still up and going strong! While also overlooking our marketing efforts and generating organic content, for rest of the team to use!

## Singapore | Dubai | Cosmoverse | Cardano Summit 2024 🚀

If you are an investor in RING - this is a very positive news for you! The moment is here! You will see Darwinia in mainstream places - we are making music (not - noise) people are coming and realizing that Darwinia is up and going strong - we have been seeing people IRL! If you are in any of the future conferences (list to be published soon!) please come say hi! We LOVE connecting people - tell us your hobbies - tell us what you would like us to do - and if you'd like to see us in the ocean, rooftops, bars, snooker clubs, whatever! We are excited and pushing forward to come see you in all of the major Crypto Hubs!

This October has been 98.6297% focused on these efforts! Please read the report [here](https://github.com/orgs/dcdao/discussions/191) I have spent countless hours - curating details, researching events, spaces and people, and then creating connections, and then warming them up followed by people maintenance! 

I am very excited about Darwinia getting exposure to the outside world and it was an enlightening moment to not only attend the event but also get invited to speak and share the panel with Osmosis, Levana and FP Block!

![image](https://github.com/user-attachments/assets/4b811ad5-e94f-4f59-8216-4357e3620135)

## Darwinia Videos

I have started making these YT shorts to work on getting more engagement- check it out [here](https://www.youtube.com/shorts/l2c38YctS5o) and please share feedback!

![image](https://github.com/user-attachments/assets/ef970440-27c8-4bb3-b041-e9ae3d0fe184)

You can view all videos on our YouTube channel [here](https://www.youtube.com/@darwinianetwork).

## Vote Ring DAO Final Version

After finishing and pushing the refactored code targeting to fix the new design for the Ring DAO website, the team reported several problems. First, we started working on each issue but it did not work very well and after fixing each report most new bugs and issues were reported. So to manage all of these, we decided to fully refactor the code with new logic and function to match the Ring DAO contract addresses. After this major task which lasted for about two weeks all the different issues were closed. And the new version is completely ready for production. According to all these things, list of the PRs and issues before refactoring are:

#### Pre-Refactor PRs and Issues:
- **PRs:**
  - [Add Darwinia Koi and Crab chain #20
](https://github.com/ringecosystem/vote.ringdao.com/pull/20)
  - [Fix logo and title when connected to Crab #24
](https://github.com/ringecosystem/vote.ringdao.com/pull/24)
- **Closed Issues:**
  - [Issue #13](https://github.com/ringecosystem/vote.ringdao.com/issues/13)

- **PR:**
  - [Support switch chain #22
](https://github.com/ringecosystem/vote.ringdao.com/pull/22)
- **Closed Issue:**
  - [Balance display issue #21
](https://github.com/ringecosystem/vote.ringdao.com/issues/21)

#### Post-Refactor Issues and PR:
The following new issues were identified and closed after the refactor:

- Issues In-Progress:
  - [Unable to update the avatar #35
](https://github.com/ringecosystem/vote.ringdao.com/issues/35)
  - [Proposal detail page display issues #34
](https://github.com/ringecosystem/vote.ringdao.com/issues/34)
  - [#33](https://github.com/ringecosystem/vote.ringdao.com/issues/33)
  - [#32](https://github.com/ringecosystem/vote.ringdao.com/issues/32)
  - [#31](https://github.com/ringecosystem/vote.ringdao.com/issues/31)
  - [#30](https://github.com/ringecosystem/vote.ringdao.com/issues/30)
  - [#29](https://github.com/ringecosystem/vote.ringdao.com/issues/29)
  - [#28](https://github.com/ringecosystem/vote.ringdao.com/issues/28)
  - [#27](https://github.com/ringecosystem/vote.ringdao.com/issues/27)
  - [#26](https://github.com/ringecosystem/vote.ringdao.com/issues/26)

-  PR:
  - [PR #26](https://github.com/ringecosystem/vote.ringdao.com/pull/26)

Community, feel free to dive into these resources and report any issues on the relevant repositories:

- [RingDAO](https://github.com/ringecosystem/vote.ringdao.com)
- [ParaLink](https://github.com/ringecosystem/paralink-ui/)

**Comment by hackfisher** [comment]
Thank you for your report. I have a proposal. In addition to regular media marketing, community communication, and brand promotion, could we organize the DCDAO team to conduct a comprehensive study and research on the market, and produce/deliver some insights and communicated content, which helps the community reach social consensus?

Key areas of focus include:
	1.	Chain Abstraction
	2.	Account Abstraction
	3.	Cross-Chain Application Fields

My sense of direction comes from some changes in the industry (particularly the development experience of Gnosis DAO). The project should shift to focus primarily on RingDAO, integrating existing cross-chain facilities and services with a cautious and investment-oriented approach. We should adopt a “Fat App + Super Dev” strategy, carefully and strategically selecting application projects to focus on.

Current ecosystem apps which meet fit this strategy might include: Paralink, Helixbox, XAccount (and its integration with governance).

Some other players working in similar area: LayerZero, Wormhole, Axelar, and Hashi.

Please note: Our resources (talent and capital) are very limited. These resources should be invested in applications and integration rather than homogeneous infrastructure (infrastructure will require external investment or sponsorship).

**Reply by hackfisher** [reply]
Refer https://wormhole.com/blog/wormhole-tally-and-scopelift-announce-multigov-the-first-ever-multichain
